tutorials/asr/cloud/aws/SageMaker_ASR_Fine-tuning-CommonVoice.ipynb

```diff
@@ -0,0 +1,666 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1c5a60a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "prefix = \"cv_esperanto\"\n",
+    "bucket = sess.default_bucket()\n",
+    "\n",
+    "loc = sess.upload_data(path=str(data_dir), bucket=bucket, key_prefix=prefix)\n",
+    "loc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6321e3a9",
+   "metadata": {},
+   "source": [
+    "### 4. Configure the training job\n",
+    "\n",
+    "Now we configure the training job, by modifying the `config.yaml` file that is stored in our source code directory.\n",
+    "We pass relative directory paths for the data based on the SageMaker mount directory on the remote instance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ac709398",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# the output path for the model checkpoints/logs\n",
+    "output_path = \"s3://\" + sess.default_bucket() + \"/nemo-output/\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8186678f",
+   "metadata": {},
+   "source": [
+    "We setup the training configuration based on the Esperanto experiments ran [here](https://github.com/andrusenkoau/NeMo/blob/esperanto_example/docs/source/asr/examples/esperanto_asr/esperanto_asr.rst).\n",
+    "\n",
+    "If you change the number of GPUs on the instance, remember to change the values below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a4b67faa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "global_batch_size = 1024\n",
+    "n_gpus = 8\n",
+    "accumulate_grad_batches = 8\n",
+    "train_batch_size = int(global_batch_size / n_gpus / accumulate_grad_batches)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4bb61640",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "conf = OmegaConf.load(config_path)\n",
+    "\n",
+    "# data config setup\n",
+    "conf.model.train_ds.batch_size=train_batch_size\n",
+    "conf.model.train_ds.is_tarred = True\n",
+    "conf.model.train_ds.tarred_audio_filepaths = \"/opt/ml/input/data/training/train_tarred_1bk/audio__OP_0..511_CL_.tar\"\n",
+    "conf.model.train_ds.manifest_filepath = \"/opt/ml/input/data/training/train_tarred_1bk/tarred_audio_manifest.json\"\n",
+    "conf.model.validation_ds.manifest_filepath = \"/opt/ml/input/data/testing/manifests/commonvoice_dev_manifest.json.clean.mount\"\n",
+    "\n",
+    "# optimization setup\n",
+    "conf.model.optim.lr = 1e-3\n",
+    "conf.model.optim.sched.name = \"CosineAnnealing\"\n",
+    "\n",
+    "# logging/output setup\n",
+    "conf.exp_manager.exp_dir = \"/opt/ml/model/\"\n",
+    "conf.exp_manager.wandb_logger_kwargs.name = \"cv_sagemaker_8gpus\"\n",
+    "conf.exp_manager.wandb_logger_kwargs.project = \"sagemaker\"\n",
+    "conf.exp_manager.wandb_logger_kwargs.entity = 'nvidia'\n",
```

Comment:
Entity as Nvidia is for our team, probably remove it and add a comment to put user value?